chromadb/test/property/test_cross_version_persist.py

```diff
@@ -336,6 +344,10 @@ def test_cycle_versions(
         name=collection_strategy.name,
         embedding_function=not_implemented_ef(),  # type: ignore
     )
+
+    # Should be able to add embeddings
+    coll.add(**embeddings_strategy)  # type: ignore
```

Comment:
causes the pickled metadata file to be loaded, a good thing to test since the class def changed and it needs special handling